File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
        if(nums.length == 1) return nums[0];
        Arrays.sort(nums);
        int check = nums.length/2;
        for(int i=0;i<nums.length-1;i++){
            int count = 0;
            int num = nums[i];
            for(int j=i;j<nums.length;j++){
                if(num == nums[j]){
                    count++;
                }
            }
            
            if(count > check){
                return num;
            }
        }
        return 0;
    }
}

File: 0169-majority-element/0169-majority-element.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var majorityElement = function(nums) {
    let majority = 0;
    let count = 0;
    for(let i=0;i<nums.length;i++){
        if(count == 0) majority = nums[i] ;
        if(majority == nums[i]) count++;
        else count--;
    }
    return majority;
};